R/exponential.R

```diff
@@ -103,3 +103,28 @@ quantile.exponential <- function(d, p, ...) {
 
   qexp(p = p, rate = d$rate)
 }
+
+#' Fit an exponential distribution to data
+#'
+#' @param d An `exponential` object created by a call to [exponential()].
+#' @param x A vector to fit the distribution to.
+#'
+#' @family exponential distribution
+#'
+#' @return An `exponential` object.
+#' @export
+fit_mle.exponential <- function(d, x, ...) {
+  ss <- suff_stat(d, x, ...)
+  exponential(ss$sum / ss$samples)
```

Comment:
Could use `mean(x)` here. I'm happy either way.

Reply:
Was really just following Distributions.jl here

Reply:
Yeah let's keep it as is upon further thought.